subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -104,29 +110,27 @@ int fota_download_start(char *host, char *file)
 		return -EINVAL;
 	}
 
+	/* TODO: Is it possible to download modem fw with a threaded model from
```

Comment:
Are  you asking if you can download two images at the same time?

Reply:
If I have two instances of download client which downloads from two different hosts using two different socket identifiers. I could pipe one stream to modem while another stream writes to flash is the idea.

Reply:
After our offline discussion, you don't need to do this any longer right?

Reply:
Correct I'll remove it for now.